### ISSUE TITLE
Settings -> Chats throws error on Dev

### ISSUE THREAD

**Issue by EdGeraghty** [syphon-org/syphon#599]
![image](https://user-images.githubusercontent.com/20861699/155748527-15e16754-eac2-4a73-94e3-6f5c97734fcf.png)

```
======== Exception caught by widgets library =======================================================
The following ConverterError was thrown building StreamBuilder<Props>(dirty, state: _StreamBuilderBaseState<Props, AsyncSnapshot<Props>>#9df1f):
Converter Function Error: 'dart:ui/platform_dispatcher.dart': Failed assertion: line 1690 pos 15: '<optimized out>': is not true.
    
#0      _AssertionError._doThrowNew (dart:core-patch/errors_patch.dart:51:61)
#1      _AssertionError._throwNew (dart:core-patch/errors_patch.dart:40:5)
#2      new Locale (dart:ui/platform_dispatcher.dart:1690:15)
#3      Props.mapStateToProps (package:syphon/views/home/settings/settings-chats-screen.dart:355:31)
#4      ChatsSettingsScreen.build.<anonymous closure> (package:syphon/views/home/settings/settings-chats-screen.dart:27:53)
#5      _StoreStreamListenerState._mapConverter (package:flutter_redux/flutter_redux.dart:532:28)
#6      _MapStream._handleData (dart:async/stream_pipe.dart:213:31)
#7      _ForwardingStreamSubscription._handleData (dart:async/stream_pipe.dart:153:13)
#8      _rootRunUnary (dart:async/zone.dart:1434:47)
#9      _CustomZone.runUnary (dart:async/zone.dart:1335:19)
#10     _CustomZone.runUnaryGuarded (dart:async/zone.dart:1244:7)
#11     _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:341:11)
#12     _BufferingStreamSubscription._add (dart:async/stream_impl.dart:271:7)
#13     _ForwardingStreamSubscription._add (dart:async/stream_pipe.dart:123:11)
#14     _WhereStream._handleData (dart:async/stream_pipe.dart:195:12)
#15     _ForwardingStreamSubscription._handleData (dart:async/stream_pipe.dart:153:13)
#16     _rootRunUnary (dart:async/zone.dart:1434:47)
#17     _CustomZone.runUnary (dart:async/zone.dart:1335:19)
#18     _CustomZone.runUnaryGuarded (dart:async/zone.dart:1244:7)
#19     _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:341:11)
#20     _DelayedData.perform (dart:async/stream_impl.dart:591:14)
#21     _StreamImplEvents.handleNext (dart:async/stream_impl.dart:706:11)
#22     _PendingEvents.schedule.<anonymous closure> (dart:async/stream_impl.dart:663:7)
#23     _rootRun (dart:async/zone.dart:1418:47)
#24     _CustomZone.run (dart:async/zone.dart:1328:19)
#25     _CustomZone.runGuarded (dart:async/zone.dart:1236:7)
#26     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1276:23)
#27     _rootRun (dart:async/zone.dart:1426:13)
#28     _CustomZone.run (dart:async/zone.dart:1328:19)
#29     _CustomZone.runGuarded (dart:async/zone.dart:1236:7)
#30     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1276:23)
#31     _microtaskLoop (dart:async/schedule_microtask.dart:40:21)
#32     _startMicrotaskLoop (dart:async/schedule_microtask.dart:49:5)
;

The relevant error-causing widget was: 
  StoreConnector<AppState, Props> StoreConnector:file:///D:/Source/syphon/lib/views/home/settings/settings-chats-screen.dart:25:41
When the exception was thrown, this was the stack: 
#0      _StoreStreamListenerState.build.<anonymous closure> (package:flutter_redux/flutter_redux.dart:518:41)
#1      StreamBuilder.build (package:flutter/src/widgets/async.dart:442:81)
#2      _StreamBuilderBaseState.build (package:flutter/src/widgets/async.dart:124:48)
#3      StatefulElement.build (package:flutter/src/widgets/framework.dart:4870:27)
#4      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4754:15)
#5      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4928:11)
#6      Element.rebuild (package:flutter/src/widgets/framework.dart:4477:5)
#7      BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2659:19)
#8      WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:882:21)
#9      RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:363:5)
#10     SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1144:15)
#11     SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1081:9)
#12     SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:995:5)
#16     _invoke (dart:ui/hooks.dart:151:10)
#17     PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:308:5)
#18     _drawFrame (dart:ui/hooks.dart:115:31)
(elided 3 frames from dart:async)
====================================================================================================
```

**Comment by EdGeraghty** [comment]
![image](https://user-images.githubusercontent.com/20861699/155749365-827ab976-6fa2-4b02-856f-8dabe3681fad.png)

**Comment by ereio** [comment]
This may be desktop only, are you seeing this in mobile?

**Comment by EdGeraghty** [comment]
Not seeing it on mobile.

**Comment by EdGeraghty** [comment]
This will be fixed with #520
